PyGObject is installed to a different prefix, so use the /S/L/E/ entry instead
--- pygtk-2.16.0-old/pygtk-codegen-2.0.in	2010-08-28 16:56:18.000000000 -0500
+++ pygtk-2.16.0/pygtk-codegen-2.0.in	2010-08-28 16:56:53.000000000 -0500
@@ -2,7 +2,7 @@
 
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-pygobject_script=@bindir@/pygobject-codegen-2.0
+pygobject_script=@%Compile_goboExecutables%@/pygobject-codegen-2.0
 
 echo "note: pygtk-codegen-2.0 is deprecated, use pygobject-codegen-2.0 instead"   > /dev/stderr
 echo "note: I will now try to invoke pygobject-codegen-2.0 in the same directory" > /dev/stderr
